### ISSUE TITLE
Package appear to be invalid

### ISSUE THREAD

**Issue by ElucGeek** [KoalaSat/pokey#64]
When I try to install pokey-arm64-v8a-v0.1.2-alpha.apk on my GrapheneOS Pixel 7 I got the error: package appear to be invalid. I try from Obtainium or manually, both fail.

**Comment by KoalaSat** [comment]
This is an issue happening since [v0.1.1-alpha](https://github.com/KoalaSat/pokey/releases/tag/v0.1.1-alpha) with Obtanium with users who installed previous versions. Since that version I started publishing specific architecture APKs and it seems that obtanium is not able to manage that change.

Just uninstall Pokey and install again the latest version using Obtanium.

**Comment by ElucGeek** [comment]
Yes the issue was already in v0.1.1, I did uninstall, reinstall from Obtainium, same error. Then I download the apk v8a install outside Obtainium, same error. Then install the universal apk 0.1.1, it works, then update in Obtainium, same error. Uninstall, download apk v.0.1.2 v8a, same error. Install the unicersal apk, it's ok but v8a should be working and Obtainium force to this version without asking.

**Comment by KoalaSat** [comment]
That seems to be related to an error on Obtanium then
